### DISCUSSION TITLE
UMEP URock : errors during wind solver equation iterations

### DISCUSSION THREAD

**Discussion by pdugue** [UMEP-dev/UMEP#563] [Q&A]
Hi,
Again a question from me !^^
I am using URock for another study case and one error appears again and again.
In the final processor step calculation, when solving the wind equations, this error came :
![image](https://github.com/UMEP-dev/UMEP/assets/138692633/548602ca-1850-4f90-bcbf-add1b3c5ede6)
I don't understand why.
Thanks for help,
Yours sincerely,
Pauline

**Comment by j3r3m1** [comment]
Do you have the right to write in the path you have hidden on your image ? Have you tried to write in a temporary folder ?

**Reply by pdugue** [reply]
Thanks for your quick answer.
It works with a temporary folder, so it's not a big issue, but we didn't have this error on the same computer until now so it was surprising.

**Reply by j3r3m1** [reply]
Have you tried on different locations on your computer ? I mean different from your temporary folder ?
